common/models/user.js

```diff
@@ -577,6 +578,15 @@ module.exports = function(User) {
         return cb(err);
       }
 
+      var AccessToken = UserModel.relations.accessTokens.modelTo;
+      AccessToken.find({ where: { userId: user.id }}, function(err, tokens) {
+        tokens.forEach(function(token) {
```

Comment:
Querying a list of tokens and then deleting them one by one is rather suboptimal. Please use bulk-delete instead.

```js
AccessToken.deleteAll({ userId: user.id }, function(err) {
  // etc.
});
```